packages/nodes-base/nodes/ProfitWell/CompanyDescription.ts

```diff
@@ -20,6 +20,7 @@ export const companyOperations: INodeProperties[] = [
 				name: 'Get Settings',
 				value: 'getSetting',
 				description: 'Get your companys ProfitWell account settings',
+				action: 'Get Settings for a company',
```

Comment:
Get settings for your company? This looks to apply to the authorized company only.